tools/topology/topology2/include/common/common_definitions.conf

```diff
@@ -27,8 +27,10 @@ Define {
 	CHANNEL_MAP_INVALID			0xFFFFFFFF
 
 	# Sample types
-	SAMPLE_TYPE_MSB_INTEGER			0 # integer with Most Significant Byte first
-	SAMPLE_TYPE_LSB_INTEGER			1 # integer with Most Significant Byte first
+	SAMPLE_TYPE_MSB_INTEGER		0 # integer with valid samples in most significant
+						  # bits, aka left aligned (not endianness!)
+	SAMPLE_TYPE_LSB_INTEGER		1 # integer with valid samples in least significant
+						  # bits, aka right aligned (not endianness!)
```

Comment:
I would keep the indentation of the comment lines consistent

Reply:
Hmm, it looks like this in emacs
```
»       # Sample types                                                                                                                                                                       
»       SAMPLE_TYPE_MSB_INTEGER»        »       0 # integer with valid samples in most significant                                                                                           
»       »       »       »       »       »         # bits, aka left aligned (not endianness!)                                                                                                 
»       SAMPLE_TYPE_LSB_INTEGER»        »       1 # integer with valid samples in least significant                                                                                          
»       »       »       »       »       »         # bits, aka right aligned (not endianness!)    
```

But bad in vi (and github). Let me add an extra space to keep both camps happy.

Reply:
Done, V2 uploaded.

Reply:
@kv2019i when are those definitions required? Is this only for the 24 valid bits in a 32-bit container?

Reply:
@plbossart Whenever the number of valid bits is different than sample containers size. But yes, in practise has impact only with 24-in-32bits.